### ISSUE TITLE
Add dynamic configuration functionality

### ISSUE THREAD

**Issue by porkbarrel** [mzuccaroli/angular-google-tag-manager#135]
Hello I would like to submit a PR that provides the option of setting the config dynamically using the APP_INITIALIZER.

Or use the `APP_INITIALIZER`

```
import { GoogleTagManagerConfiguration } from 'angular-google-tag-manager-config.service';

imports: [
    ...
    GoogleTagManagerModule.forRoot()
]

providers: [
    {
      ...
      provide: APP_INITIALIZER,
      useFactory: configInitializer,
      deps: [
        HttpBackend,
        GoogleTagManagerConfiguration,
      ],
      multi: true,
    },
  ],
```

set the config in the method assigned to useFactory

```
googleTagManagerConfiguration.set(googleTagManagerConfiguration);
```

**Comment by porkbarrel** [comment]
PR submitted

**Comment by mzuccaroli** [comment]
will test your PR asap!

**Comment by mzuccaroli** [comment]
your PR has benn merged & published in version 1.5.1
thanks for the contribution
